etc/lore.config

```diff
@@ -3,3 +3,5 @@ default_ui=dialog
 dialog_layout=black_and_white
 lists=
 download_to=/tmp/kw_download
+kernel_tree=
+target_branch=
```

Comment:
For some reason, when I introduced this file, I did not add proper documentation. Could you create one commit, adding the documentation to the previous and new options? See the below link as a reference:

https://github.com/kworkflow/kworkflow/blob/unstable/etc/mail.config

Reply:
Also, let's call `kernel_tree_path`.